### ISSUE TITLE
Transfer Learning

### ISSUE THREAD

**Issue by bmfarer** [PaddlePaddle/PaddleDetection#3508]
请问一下开发者大大下一个版本能够增加迁移学习方面的灵活性吗？因为现在就几个参数，在做小样本迁移学习的时候感觉有点回天乏术(〒▽〒)

* 说明：

1. pretrain\_weights的路径为COCO数据集上开源的faster RCNN模型链接，完整模型链接可参考[MODEL_ZOO](https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.1/static/docs/advanced_tutorials/TRANSFER_LEARNING_cn.md)
2. finetune\_exclude\_pretrained\_params中设置参数字段，如果参数名能够匹配以上参数字段（通配符匹配方式），则在模型加载时忽略该参数。

如果用户需要利用自己的数据进行finetune，模型结构不变，只需要忽略与类别数相关的参数，不同模型类型所对应的忽略参数字段如下表所示：</br>

|      模型类型      |             忽略参数字段                  |
| :----------------: | :---------------------------------------: |
|     Faster RCNN    |          cls\_score, bbox\_pred           |
|     Cascade RCNN   |          cls\_score, bbox\_pred           |
|       Mask RCNN    | cls\_score, bbox\_pred, mask\_fcn\_logits |
|  Cascade-Mask RCNN | cls\_score, bbox\_pred, mask\_fcn\_logits |
|      RetinaNet     |           retnet\_cls\_pred\_fpn          |
|        SSD         |                ^conv2d\_                  |
|       YOLOv3       |              yolo\_output                 |

**Comment by qingqing01** [comment]
@FallBlackMagic 

静态图的的迁移学习，一种方式是直接加载，shape不匹配的参数可以直接忽略，请看文档：

<img width="957" alt="图片" src="https://user-images.githubusercontent.com/7845005/123515778-679d4f80-d6cb-11eb-9984-201298919898.png">

**Comment by FallBlackMagic** [comment]
@qingqing01 嗯嗯，主要是我想实时别的迁移学习方法，但是里面似乎没有合适的参数可以设置

**Comment by bmfarer** [comment]
@qingqing01 嗯嗯，主要是我想实现别的迁移学习方法，但是里面似乎没有合适的参数可以设置

**Comment by qingqing01** [comment]
@FallBlackMagic  您如果是一些参数能加载的也想忽略，可以把模型里的参数名字打印看下，依据名字指定就行。
